chore(docs): removing UDP reference (#1048)
questdb/questdb.io: docs/reference/configuration.md

  - The environment variable `QDB_CAIRO_SQL_COPY_ROOT`.
  - The `cairo.sql.copy.root` in `server.conf`.
- The path for the source CSV file is mounted.
- The source CSV file path and the path defined by `QDB_CAIRO_SQL_COPY_ROOT` are
  identical.
- It is optional to define `QDB_CAIRO_SQL_COPY_WORK_ROOT`.

The following is an example command to start a QuestDB instance on Docker, in
order to import a CSV file:

```shell
docker run -p 9000:9000 \
-v "/tmp/questdb:/var/lib/questdb" \
-v "/tmp/questdb/my_input_root:/var/lib/questdb/questdb_import" \
-e QDB_CAIRO_SQL_COPY_ROOT=/var/lib/questdb/questdb_import \
questdb/questdb
```

Where:

- `-v "/tmp/questdb/my_input_root:/var/lib/questdb/questdb_import"`: Defining a
  source CSV file location to be `/tmp/questdb/my_input_root` on local machine
  and mounting it to `/var/lib/questdb/questdb_import` in the container.
- `-e QDB_CAIRO_SQL_COPY_ROOT=/var/lib/questdb/questdb_import`: Defining the
  copy root directory to be `var/lib/questdb/questdb_import`.

> It is important that the two path to be identical

Reviewer: ```suggestion
It is important that the two paths are identical
```

Reviewer 2: good one! it fact there is a more serious issue in this part: a leading slash is missing in 2 cases: 
<img width="849" alt="image" src="https://user-images.githubusercontent.com/158619/191504591-02cd36d8-8090-4b14-8f42-bee5c49812e1.png">

Let me push a new commit
